sinfonier-web/app/controllers/Topologies.java

```diff
@@ -122,10 +122,27 @@ public static void remove(String id) throws SinfonierException {
     if (topology != null) {
       if (!topology.isRunning()) {
         topology.remove();
-        index(null, null, 1);
+        if (request.isAjax()) {
```

Comment:
This call is make from "Delete" button of editor, from "deleteWiring" function defined on adapter.js